django-stubs/contrib/postgres/forms/ranges.pyi

```diff
@@ -1,17 +1,24 @@
 from django import forms
 from django.forms.widgets import MultiWidget
-from typing import Any, Callable, Optional
+from typing import Any, Callable, Optional, Type, Union
 
 _OptAttrs = Dict[str, Any]
 
 class BaseRangeField(forms.MultiValueField):
     def __init__(self, **kwargs: Any) -> None: ...
     def compress(self, values: Any) -> Tuple[Optional[Callable], Optional[Callable]]: ...
 
-class IntegerRangeField(BaseRangeField): ...
-class FloatRangeField(BaseRangeField): ...
-class DateTimeRangeField(BaseRangeField): ...
-class DateRangeField(BaseRangeField): ...
+class IntegerRangeField(BaseRangeField):
+    base_field: Union[Type[forms.Field], forms.Field] = ...
```

Comment:
`base_field` defined on `forms.Field`, I think. If it may be both `Type[Field]` and `Field`, better fix it there, and remove all this duplication.

Reply:
addressed in 67d664d